virttest/utils_test/libvirt.py

```diff
@@ -892,7 +892,7 @@ class PoolVolumeTest(object):
     """Test class for storage pool or volume"""
 
     def __init__(self, test, params):
-        self.tmpdir = data_dir.get_tmp_dir()
+        self.base_dir = data_dir.get_data_dir()
```

Comment:
If change the class variable name it will affect the usage for it. I suggest do not change the variable name, or you need to update all usage for self.tmpdir

Reply:
@kylazhang how about instead of replacing, I keep tmpdir, and add a new var base_dir?

Reply:
I think just keep tmpdir is ok, no need extra var

Reply:
@kylazhang Thank you for the review, I'm running a job to verify the change. I'll update the result as soon as I get it.